src/main/java/it/pagopa/transactions/mdcutilities/TracingUtils.java

```diff
@@ -0,0 +1,97 @@
+package it.pagopa.transactions.mdcutilities;
+
+import it.pagopa.ecommerce.commons.domain.v1.RptId;
+import it.pagopa.ecommerce.commons.domain.v1.TransactionId;
+import reactor.util.context.Context;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Tracing utility class that contains helper methods to set transaction
+ * information, such as transactionId and rptId list into reactor context
+ */
+public class TracingUtils {
```

Comment:
The name clash with the `TracingUtils` class inside commons makes usage a bit clunky.
What about `LabelTracingUtils`, `SpanLabelTracingUtils`, `SpanLabelUtils` or `MDCTracingUtils` (I like this one less)?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/289/commits/88eb6998da70004e6f5c3ab259e9a30b6e430614 refactoring to `TransactionTracingUtils`